### PR TITLE
Added eslint import resolver

### PR DESCRIPTION
**Pull request by N0bl3**
Added eslint import resolver in devDependencies and removed inexact eslint rules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "eslint": "^2.3.0",
     "eslint-config-airbnb": "^6.1.0",
     "eslint-plugin-meteor": "^3.0.1",
-    "eslint-plugin-react": "^4.0.0"
+    "eslint-plugin-react": "^4.0.0",
+    "eslint-import-resolver-meteor": "*"
```
Comment (tmeasday) [inline package.json]: Should we constrain the version a little?

**Comment by tmeasday** [comment]
See #132

**Comment by N0bl3** [comment]
Oops didn't see it. We can discard this PR... No idea how to do it though :)

**Comment by hwillson** [comment]
Closing this off as these changes are already in place. Thanks!

**Pull request closed without merging**